sdk/communication/AzureCommunicationCalling/Swift_Generated_Interfaces/AzureCommunicationCalling.h

```diff
@@ -0,0 +1,1124 @@
+
+    
+    /// HangUp a call
+    open func hangUp(options: HangUpOptions!, completionHandler: ((Error?) -> Void)!)
+
+    
+    /// Remove a participant from a call
+    open func remove(participant: RemoteParticipant!, completionHandler: ((Error?) -> Void)!)
+
+    
+    /// Hold this call
+    open func hold(completionHandler: ((Error?) -> Void)!)
+
+    
+    /// Resume this call
+    open func resume(completionHandler: ((Error?) -> Void)!)
+
+    
+    // Class extension begins for Call.
+    open var callerId: CommunicationIdentifier
+
+    open func add(participant: CommunicationIdentifier!) -> RemoteParticipant!
+
+    open func add(participant: PhoneNumberIdentifier!, options: AddPhoneNumberOptions!) -> RemoteParticipant!
+}
+
+// Class extension ends for Call.
+
+
+/// Describes a remote participant on a call
+open class RemoteParticipant : NSObject {
+
+    
+    /// Deallocates the memory occupied by this object.
+    open func dealloc()
+
+    
+    /// Private Preview Only: Display Name of the remote participant
+    open var displayName: String! { get }
+
+    
+    /// True if the remote participant is muted
+    open var isMuted: Bool { get }
+
+    
+    /// True if the remote participant is speaking
+    open var isSpeaking: Bool { get }
+
+    
+    /// Reason why participant left the call, contains code/subcode.
+    open var callEndReason: CallEndReason! { get }
+
+    
+    /// Current state of the remote participant
+    open var state: ParticipantState { get }
+
+    
+    /// Remote Video streams part of the current call
+    open var videoStreams: [RemoteVideoStream]! { get }
+
+    
+    /**
+     * The delegate that will handle events from the ACSRemoteParticipant.
+     */
+    unowned(unsafe) open var delegate: RemoteParticipantDelegate!
+
+    
+    // Class extension begins for RemoteParticipant.
+    open var identity: CommunicationIdentifier
+}
+
+// Class extension ends for RemoteParticipant.
+
+
+/// Describes the reason for a call to end
+open class CallEndReason : NSObject {
+
+    
+    /// Deallocates the memory occupied by this object.
+    open func dealloc()
+
+    
+    /// The code
+    open var code: Int32 { get }
+
+    
+    /// The subcode
+    open var subcode: Int32 { get }
+}
+
+
+/// Video stream on remote participant (NOT SUPPORTED)
+open class RemoteVideoStream : NSObject {
+
+    
+    /// Deallocates the memory occupied by this object.
+    open func dealloc()
+
+    
+    /// True when remote video stream is available.
+    open var isAvailable: Bool { get }
+
+    
+    /// MediaStream type of the current remote video stream (Video or ScreenShare).
+    open var type: MediaStreamType { get }
+
+    
+    /// Unique Identifier of the current remote video stream.
+    open var id: Int32 { get }
+}
+
+
+/// Describes a PropertyChanged event data
+open class PropertyChangedEventArgs : NSObject {
+
+    
+    /// Deallocates the memory occupied by this object.
+    open func dealloc()
+}
```

Comment:
Does this have no info on what property changed?

Reply:
Maybe in the wrong place, can you clarify ?

Reply:
Lines 836-841... has no info on what property changed. It seems to have no properties at all.